src/Controller/BlockAdminController.php

```diff
@@ -86,10 +86,6 @@ public function createAction(Request $request): Response
     {
         $this->admin->checkAccess('create');
 
-        if (!$this->admin->isChild()) {
```

Comment:
This check was modified by me on a previous PR. IMO we can remove it. Create action is executable on by child admin OR by shared block admin.

Reply:
But the remaining one (BlockAdmin) does not have the create route, I have remove it.